src/sections/references.html

```diff
@@ -0,0 +1,35 @@
+<section class="flex flex-col items-end pl-5 pr-10 my-10 bg-no-repeat bg-cover center sm:px-10 lg:px-40" style="background-image: url(./images/ref-yellow-shape.svg);">
+
+    <ul class="flex flex-col items-end gap-[2px] py-10 text-xl font-medium md:text-2xl md:py-44">
+        <li class="cursor-pointer">Ace Hotel</li>
+        <li class="cursor-pointer">ARMEDANGELS</li>
+        <li class="cursor-pointer">BERLIN DESIGN WEEK</li>
+        <li class="cursor-pointer">Berliner Berg</li>
+        <li class="cursor-pointer">BITE CLUB</li>
+        <li class="cursor-pointer">Budweiser Budvar</li>
+        <li class="cursor-pointer">ChungKing Noodles</li>
+        <li class="cursor-pointer">Designpreis Brandenburg</li>
+        <li class="cursor-pointer">Die Techniker</li>
+        <li class="cursor-pointer">European Street Food Awards</li>
+        <li class="cursor-pointer">Grundmann Dentistry</li>
+        <li class="cursor-pointer">HORNBACH Werkstück</li>
+        <li class="cursor-pointer">HORNBACH macht Schule</li>
+        <li class="cursor-pointer">iF Design</li>
+        <li class="cursor-pointer">Kopka</li>
+        <li class="cursor-pointer">KLH Maßschuhe</li>
+        <li class="cursor-pointer">Lode & Stijn</li>
+        <li class="cursor-pointer">Luya</li>
+        <li class="cursor-pointer">MEISSEN</li>
+        <li class="cursor-pointer">Motel Beer & Coffee</li>
+        <li class="cursor-pointer">PAPER & TEA</li>
+        <li class="cursor-pointer">rocket & basil</li>
+        <li class="cursor-pointer">Seeberger Gruppe</li>
+        <li class="cursor-pointer">Shiori</li>
+        <li class="cursor-pointer">SPOC Magazin</li>
+        <li class="cursor-pointer">stocubo</li>
+        <li class="cursor-pointer">STUR</li>
+        <li class="cursor-pointer">World of Coffee</li>
+        <li class="cursor-pointer">yamo</li>
+    </ul>
```

Comment:
While using plain HTML to render a large list may not cause significant issues, it is not considered best practice. Instead, it is better to use JavaScript for rendering such lists.
```suggestion
    <ul id="dynamic-lists" class="flex flex-col items-end gap-[2px] py-10 text-xl font-medium md:text-2xl md:py-44">
       
    </ul>
```
in the `index.html` file use javascript by storing the list items into an array and append each list item on the id='dynamic-lists' 

```js
 
const ul = document.getElementById('dynamic-lists')

 if (id === references) {
        const listItems = [
                "Ace Hotel",
                "ARMEDANGELS",
                "BERLIN DESIGN WEEK",
                "Berliner Berg",
                "BITE CLUB" 
                  ...,
           ]

      listItems.map(item =>  // render and append the listItems)
```

This approach enhances scalability, maintainability, and flexibility, especially if the list content needs to be dynamic or frequently updated.